lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -168,18 +158,6 @@ def __init__(
         # for bounding box calculations
         self.spines[:].set_visible(False)
 
-        if auto_add_to_figure:
```

Comment:
The message said this would raise for non-true values rather than raise if it is passed.  I think we need to keep supporting this as a kwarg for a while to be kind to people trying to support a wide range of mpl versions.